README.md

```diff
@@ -1,6 +1,6 @@
 [![Logo][jhipster-image]][jhipster-url]
 
-[![NPM version][npm-image]][npm-url] [![Downloads][npmcharts-image]][npmcharts-url] [![Gitter][gitter-badge-image]][gitter-badge-url] [![Known Vulnerabilities][snyk-image]][snyk-url] [![Package Health][health-image]][health-url]
+[![NPM version][npm-image]][npm-url] [![Downloads][npmcharts-image]][npmcharts-url] [![Gitter][gitter-badge-image]][gitter-badge-url] [![Known Vulnerabilities][snyk-image]][snyk-url] [![Package Health][health-image]][health-url] [![Gurubase](https://img.shields.io/badge/Gurubase-Ask%20JHipster%20Guru-006BFF)](https://gurubase.io/g/jhipster)
 
 [![Generator Build Status][github-actions-generator-image]][github-actions-url] [![Angular Build Status][github-actions-angular-image]][github-actions-url] [![React Build Status][github-actions-react-image]][github-actions-url] [![Vue Build Status][github-actions-vue-image]][github-actions-url] [![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.jhipster.tech/scans)
 
```

Comment:
Can you please move this badge to its own 3rd row so it doesn't wrap? Also, I think a sentence about JHipster Guru inline would be good. I tried to edit this file on your branch, but was unable to. Here's what I think the first 11 lines should be:

```
[![Logo][jhipster-image]][jhipster-url]

[![NPM version][npm-image]][npm-url] [![Downloads][npmcharts-image]][npmcharts-url] [![Gitter][gitter-badge-image]][gitter-badge-url] [![Known Vulnerabilities][snyk-image]][snyk-url] [![Package Health][health-image]][health-url] 

[![Generator Build Status][github-actions-generator-image]][github-actions-url] [![Angular Build Status][github-actions-angular-image]][github-actions-url] [![React Build Status][github-actions-react-image]][github-actions-url] [![Vue Build Status][github-actions-vue-image]][github-actions-url] [![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.jhipster.tech/scans)

[![Gurubase](https://img.shields.io/badge/Gurubase-Ask%20JHipster%20Guru-006BFF)](https://gurubase.io/g/jhipster)

Greetings, Java Hipster!

Full documentation and information is available on our website at [https://www.jhipster.tech][jhipster-url]. You can also [ask JHipster Guru](https://gurubase.io/g/jhipster) if you want to use AI to answer your JHipster questions.
```